ci/Makefile

```diff
@@ -2,7 +2,17 @@
 .PHONY: setup-conda-env build-hail-ci push-hail-ci test-locally
 .PHONY: run-service deploy
 
-PROJECT = $(shell gcloud config get-value project)
+PROJECT=${shell gcloud config get-value project}
+
+# Get unique instance name, external_ip, and zone, to avoid hardcoding vars
+# There may be a better way of doing this, but simply ${ARR[N]} did not work
```

Comment:
`${ARR[N]}` looks like POSIX shell syntax to me. Did you try `$(shell bash -c "${ARR[N]}")`?

Reply:
I think the issue is that I wasn't using single-evaluation variables ( := ). I'll try again.

Reply:
I tried both ways, including $(shell bash ..) and no luck. This worked!

```makefile
INSTANCE:=${word 1, ${DETAILS}}
EXTERNAL_IP:=${word 4, ${DETAILS}}
ZONE:=${word 5, ${DETAILS}}
```

Reply:
Ah, you can't use bash array syntax on things that are not bash arrays. A bash array is a parentheses delimited sequence of values, i.e.:
```
(a b c d e f)
```
The output of `gcloud compute instances list` is lines of space delimited text. There are ways to convert this to a bash array using `read` and the `IFS` variable, but, IMO, using awk is more straightforward.